src/main/java/ch/uzh/ifi/hase/soprafs21/objects/GameEngine.java

```diff
@@ -326,4 +328,33 @@ public Player findPlayerbyUsername(Game game, String playerName){
         }
         return null;
     }
+
+    public void createGameFromGameSessionAndFillUp(GameSession gameSession){
+
+        int numberOfUsersToInvite = 4 - gameSession.getUserList().size();
```

Comment:
würde diese 4 und die vom createGameFromGameSession(GameSession gameSession) als private static final int PLAYER_AMOUNT oder so umschreiben, ist ja die gleiche logik

Reply:
erledigt. allerdings frage. gibt es einen grund, dass sie static sein muss? für potencial future ussage ausserhalb von gameengine thats it?